tools/build.py

```diff
@@ -191,9 +191,9 @@ def init_options():
         action='store_true', default=False,
         help='Enable JerryScript heap statistics')
     jerry_group.add_argument('--jerry-profile',
-        metavar='FILE', action='store', default='es5.1',
+        metavar='FILE', action='store', default='es.next',
```

Comment:
Open question: shall the bump to JerryScript 2.3 also include a bump in the supported ES version?

Reply:
Ohh. I've missed this. I've just wanted to change the comment below. Will update this asap to keep it at es5.1 for now.